sdk/storage/Directory.Build.props

```diff
@@ -54,7 +54,7 @@
     <DefineConstants>DEBUG;$(DefineConstants)</DefineConstants>
   </PropertyGroup>
 
-  <Import Condition="'$(IsClientLibrary)' == 'true' AND '$(IsMgmtClientLibrary)' != 'true'" Project="$(MSBuildThisFileDirectory)..\core\Azure.Core\src\Azure.Core.props" />
+  <Import Condition="'$(IsClientLibrary)' == 'true' AND '$(IsMgmtClientLibrary)' != 'true' AND '$(IsWebjobsProject)' != 'true'" Project="$(MSBuildThisFileDirectory)..\core\Azure.Core\src\Azure.Core.props" />
```

Comment:
Can we just move this reference to individual projects that need it?

Reply:
Looks like `Azure.Storage.Common` uses Core types and anything else within Storage is using that. So I moved it there.